lib/state/settings/actions.ts

```diff
@@ -49,6 +49,15 @@ export const setLineLength: A.ActionCreator<A.SetLineLength> = (
   lineLength,
 });
 
+export const toggleDisableHotkeys = () => (dispatch, getState) => {
+  const newValue = !getState().settings.disableHotkeys;
```

Comment:
For simplicity and consistency with the rest of the actions, I don't think we need to set a variable here, we can just pass the value below like `sortReversed` etc. is doing.